src/linter/ui5Types/SourceFileLinter.ts

```diff
@@ -1241,7 +1247,7 @@ export default class SourceFileLinter {
 	 * from there. Directly finding the type of the property is not possible from here as it's
 	 * missing some context.
 	*/
-	#isPropertyBinding(node: ts.NewExpression | ts.CallExpression, propName: string | undefined) {
+	#isPropertyBinding(node: ts.NewExpression | ts.CallExpression, propName: string[] | undefined) {
```

Comment:
I think, with the new line 1182, the `propName` (array) can no longer be `undefined`. But an entry in the array can be the empty string (which then shouldn't match anything)

Reply:
Ah, thanks!